modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * assembly/registry
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-configuration-resource-overview-{context}"]
+= Image Registry Operator configuration
+
+The image-registry resource offers the following configuration fields.
+
+[cols="3a,8a",options="header"]
+|===
+|Component |Description
```

Comment:
Are these fields, or components, or states? It feels like the descriptions have different functions for different entries. I feel like some of them would benefit from having values or default values provided.